tests/fixtures/network/nmstate.py

```
@@ -0,0 +1,31 @@
+from utilities.constants.namespaces import NamespacesNames
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
+def nmstate_namespace(admin_client):
+    try:
+        return Namespace(client=admin_client, name=NamespacesNames.OPENSHIFT_NMSTATE, ensure_exists=True)
+
+    except ResourceNotFoundError:
+        LOGGER.info(f"Namespace '{NamespacesNames.OPENSHIFT_NMSTATE}' not found.")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)